### DISCUSSION TITLE
Pull through cache : untrusted remote registry?

### DISCUSSION THREAD

**Discussion by amitkumar2283** [distribution/distribution#4384] [General]
We are planning to use Distribution to act as Pull Through Cache for  a remote-registry (based on Nexus).
This is a  secure registry with SSL certs issued by `issuer: C=US; O=Amazon; CN=Amazon RSA 2048 M02`

Unfortunately, Distribution failed to verify the certificate of this remote-registry 

````
panic: Get "https://remote-registry/v2/": tls: failed to verify certificate: x509: certificate signed by unknown authority
````

Can we pass on the CA via configuration to Distribution? OR can we update the keystone of distribution?

**Comment by milosgajdos** [comment]
You can pass CA certs via config https://distribution.github.io/distribution/about/configuration/#tls

**Reply by amitkumar2283** [reply]
Thanks for the response.
Would this certificate be used for both the Distribution server and the remote-registry?
